StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentElementTest.swift

```diff
@@ -196,7 +196,7 @@ class EmbeddedPaymentElementTest: XCTestCase {
         sut.embeddedPaymentMethodsView.didTap(rowButton: sut.embeddedPaymentMethodsView.getRowButton(accessibilityIdentifier: "Cash App Pay"))
         // ...updating w/ a broken config...
         let brokenConfig = EmbeddedPaymentElement.IntentConfiguration(mode: .payment(amount: -1000, currency: "bad currency"), confirmHandler: { _, _, _ in })
-        async let _ = sut.update(intentConfiguration: brokenConfig)
+        _ = await sut.update(intentConfiguration: brokenConfig)
```

Comment:
What's the reason behind this change?

Reply:
We may actually have a threading problem on master! In this test if you change it from an invalid config to a valid one, the code just deadlocks. I think it may be due to the test code but this shouldn't occur.